pegasus/sites.v3/code.org/styles/040-page.css

```diff
@@ -493,14 +493,6 @@ th a:active, th a:hover, th a:link, th a:visited {
   color: #c1c1c1 !important
 }
 
-.btn-carousel-slide {
```

Comment:
These are left over from [the PR](https://github.com/code-dot-org/code-dot-org/pull/58518) that deprecated all of this on the code.org site.